deploy/base/model.yaml

```diff
@@ -1 +1 @@
-embeddings_file: embeddings_us_filtered_2023_03_23_17_22_22.csv
\ No newline at end of file
+embeddings_file: embeddings_us_filtered_2023_03_24_14_26_08.csv
```

Comment:
Could you regenerate it please?  The current head is at `embeddings_us_filtered_2023_03_25_12_09_56`, we want to make sure your timestamp is after (so it includes change I made)

Reply:
updated embeddings. I was going on what was written on the issue bug, but can make a topic in follow up PR for the wealthy query.

Reply:
Sorry, I'd consider the initial bug description as a report on the issue.  Couple of reasons for not tying "wealthy neighborhoods" to Mean_Income_Person, and instead creating a topic:
1. The data coverage for Median_Income_Household/Person is better: https://datacommons.org/tools/statvar#sv=Median_Income_Household vs. https://datacommons.org/tools/statvar#sv=Mean_Income_Household
2. Wealthy neighborhoods could be based on more factors/SVs too like home price.

And maybe we could call the topic "community affluence" or something (wealthy is one part of the spectrum)?

If you want to submit this PR though, maybe remove that one description and submit?